avocado/core/result.py

```diff
@@ -81,6 +81,10 @@ def check_test(self, state):
         for output_plugin in self.output_plugins:
             output_plugin.check_test(state)
 
+    def tests_total(self, tests_total):
```

Comment:
I'd call this `set_tests_total` to emphasize it's not executing anything, it's just overriding a variable.